src/jit/optimizer.cpp

```diff
@@ -3293,7 +3293,7 @@ void Compiler::optUnrollLoops()
  *  not execute a method call.
  */
 
-bool Compiler::optReachWithoutCall(BasicBlock* topBB, BasicBlock* botBB)
+bool Compiler::optReachWithoutCall(BasicBlock* topBB, BasicBlock* botBB, bool endsWithTailCall /* = false*/)
```

Comment:
The function header comment on this method not very informative. It appears that `true` is the conservative answer, based on the code below, but it would be good to modify the comment to make that clear, e.g. "Return false only if it can be proven that there is a code path from 'topBB' to 'botBB' that will not execute a method call or hit a BBF_GC_SAFE_POINT."